### PR TITLE
Fix Issue #21

### PR DESCRIPTION
**Pull request by yokotaso**
If @kintone/dts-gen has nunchuk depencies,
Publish Size: 114kb
Install Size: 15.9MB
https://packagephobia.now.sh/result?p=%40kintone%2Fdts-gen

After remove nunchuck, npm pack --dry-run
package size:  27.8 kB
unpacked size: 127.8 kB

### CODE REVIEW

File: resources/testfield.d.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
+declare namespace kintone.types {
```
Comment (yokotaso) [inline resources/testfield.d.ts]: Type definition file generated by DemoClient

File: src/utils/logger.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export function log(message: string) {
```
Comment (koba04) [inline src/utils/logger.ts]: Why did you add this abstraction?

Comment (yokotaso) [reply src/utils/logger.ts]: fix eslint error

**Comment by koba04** [comment]
@yokotaso Awesome! Thank you 👏👏👏
